src/styles/variables.less

```diff
@@ -4,3 +4,6 @@
 
 // copied from origin-web-console
 @sidebar-os-bg: #30363d;
+
+@saas-text: @color-pf-black-800; //to be replaced with @color-pf-white when the background image is in place
+@saas-hover: fade(#09405e, 80%);
```

Comment:
We should stick to the Patternfly color palette. @color-pf-blue-600 is pretty close.